examples/jquery/README.md

```diff
@@ -1,11 +1,27 @@
 # JQuery Example
 
-This example exposes a simple service at the route `http://localhost:3000/flakeyService`. As the service receives requests, it gets slower and slower. Once it takes more than 1 second to respond, the service just returns a `423 (Locked)` error.
+This example exposes a simple service at the route `http://localhost:3000/flakeyService`. 
+For every request the service receives, the response time is increased.
+[The service returns a `423 (Locked)` error if the response time is above 1000ms.](https://github.com/bucharest-gold/opossum/blob/master/examples/jquery/index.js#L43-L46) This example also has a web frontend at `http://localhost:3000/` for interacting with the service.
 
-Start the server.
+To build and run this example:
 
-```sh
-$ npm start
-```
+1. From the root of the project, build the `opossum` module:
```

Comment:
Maybe this should say, "from the root of the opossum repository", since there are two "projects" we are dealing with: the opossum module itself, and the example project.